modules/uploads/db/migrations/20240417160058_add_multipart_i_ds_to_multipart_file_uploads/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
```

Comment:
Need to remove these migrations and recreate the previous one. This is creating a lot of excess migrations.